### PR TITLE
configuration adjustments

### PR DESCRIPTION
**Pull request by juaresba**
Makes additional adjustments to the project's configuration so it is leaner

### CODE REVIEW

File: template/vtex/tsconfig.json [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-    "typeRoots": ["node_modules/@types"],
-    "types": ["node", "jest", "graphql"],
-    "module": "esnext",
-    "moduleResolution": "node",
-    "target": "es2017",
     "rootDirs": ["src", "stories"],
     "baseUrl": "src",
   },
-  "include": ["src"],
-  "exclude": ["node_modules", "dist", "example"]
```
Comment (emersonlaurentino) [inline template/vtex/tsconfig.json:14]: this line must be maintained, just rename `example` to `stories`

File: template/vtex/package.json [hunk 2]
```diff
@@ -11,14 +11,18 @@
...
+    "develop": "microbundle-crl watch --no-compress --format modern,cjs",
+    "start": "start-storybook -p 9009",
     "build": "microbundle-crl --no-compress --format modern,cjs",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs",
-    "prepare": "run-s build",
-    "test:lint": "eslint .",
-    "storybook": "start-storybook -p 9009",
-    "build-storybook": "build-storybook"
+    "build:storybook": "build-storybook",
+    "lint": "eslint --ext .js,.tsx,.jsx,.ts .",
```
Comment (emersonlaurentino) [inline template/vtex/package.json]:
```suggestion
    "lint": "eslint src/ --ext .tsx,.ts",
```

Comment (juaresba) [reply template/vtex/package.json]: Resolved

File: template/vtex/tsconfig.json (hunk 1 above)
Comment (juaresba) [reply template/vtex/tsconfig.json:14]: Resolved

**Review by emersonlaurentino — APPROVED** [review]

**Pull request merged**